bin/dnfdragora

```diff
@@ -12,7 +12,6 @@
 
 import sys
 import gettext
-from gettext import gettext as _
```

Comment:
Don't we need this for the translated strings? We usually have our strings prefixed with `_`, like the following:

```python
parser.add_argument('--exit',        help=_('force dnfdaemon dbus services used by dnfdragora to exit'), action='store_true')
```